### PR TITLE
Template improvements

### PR DESCRIPTION
**Pull request by emanuelschuetze**
see commit messages for details

### CODE REVIEW

File: openslides/agenda/static/templates/agenda/item-list.html [hunk 1]
```diff
@@ -311,10 +325,6 @@ <h1 translate>Agenda</h1>
       <!-- main content column -->
       <div class="col-xs-6 content"
         style="padding-left: calc({{ items.length === itemsSearched.length ? item.parentCount : 0 }}*25px)">
-        <div class="icon-column">
```
Comment (FinnStutzenstein) [inline openslides/agenda/static/templates/agenda/item-list.html]: if you remove `icon-column` you also can remove the `title-column`.

Comment (FinnStutzenstein) [reply openslides/agenda/static/templates/agenda/item-list.html]: these classes are just for the purpose, that the div's are next to each other

File: openslides/agenda/static/templates/agenda/list-of-speakers-partial-management.html [hunk 2]
```diff
@@ -10,15 +10,10 @@
...
-        <i class="fa fa-toggle-off"></i>
-        <translate>Closed</translate>
-      </button>
-      <button ng-if="!item.speaker_list_closed" ng-click="closeList(true)"
-          class="btn btn-sm btn-default">
-        <i class="fa fa-toggle-on"></i>
-        <translate>Open</translate>
+      <button ng-click="closeList(!item.speaker_list_closed)"
+          class="btn btn-sm" ng-class="item.speaker_list_closed ? 'btn-danger' : 'btn-default'">
+        <i class="fa" ng-class="item.speaker_list_closed ? 'fa-square' : 'fa-square-o'"></i>
```
Comment (FinnStutzenstein) [inline openslides/agenda/static/templates/agenda/list-of-speakers-partial-management.html]: This empty box is some kind of useless.. Why not change the button text to 'open list of speakers'?

File: openslides/core/static/css/_helper.scss [hunk 3]
```diff
@@ -243,3 +247,17 @@ tr.selected td {
     background-color: #fcf8e3;
     border-color: #faebcc;
 }
+
+/* countdown */
```
Comment (FinnStutzenstein) [inline openslides/core/static/css/_helper.scss]: Why did you move this here? This should be in the _countdown.scss

File: openslides/core/static/css/core/_countdown.scss [hunk 4]
```diff
@@ -24,9 +24,3 @@
     font-weight: normal;
     font-size: 5vw;
 }
-#countdown.warning {
```
Comment (FinnStutzenstein) [inline openslides/core/static/css/core/_countdown.scss]: Please move the countdown related rules here.

File: openslides/core/static/templates/core/manage-projectors.html [hunk 5]
```diff
@@ -127,7 +129,7 @@ <h1 translate>Manage projectors</h1>
         }
 
         #iframewrapper_{{ projector.id }} {
-            height: {{ projector.iframeHeight }}px;
+            height: calc({{ projector.iframeHeight }}px + 2px);
```
Comment (FinnStutzenstein) [inline openslides/core/static/templates/core/manage-projectors.html]: can you use `{{ projector.iframeHeight + 2 }}`?

File: openslides/core/static/js/core/base.js [hunk 6]
```diff
@@ -1070,7 +1070,7 @@ angular.module('OpenSlidesApp.core', [
              *  - YOffset: move the editor toolbar 40px up
              */
             getOptions: function () {
-                var extraPluginsString = 'colorbutton,find,sourcedialog,justify,showblocks';
+                var extraPluginsString = 'colorbutton,colordialog,find,sourcedialog,justify,showblocks';
```
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: WHat does this plugin do? I do not have any new button.

File: openslides/agenda/static/templates/agenda/item-list.html (hunk 1 above)
Comment (emanuelschuetze) [reply openslides/agenda/static/templates/agenda/item-list.html]: Done

File: openslides/agenda/static/templates/agenda/list-of-speakers-partial-management.html (hunk 2 above)
Comment (emanuelschuetze) [reply openslides/agenda/static/templates/agenda/list-of-speakers-partial-management.html]: Done.

File: openslides/core/static/css/core/_countdown.scss (hunk 4 above)
Comment (emanuelschuetze) [reply openslides/core/static/css/core/_countdown.scss]: done

File: openslides/core/static/js/core/base.js (hunk 6 above)
Comment (emanuelschuetze) [reply openslides/core/static/js/core/base.js]: `colordialog` provides more colors with input for specific color values

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein All done. Please approve. - I have to merge it now.
